File: test/fixtures/output/ruby/native/application-form-encoded.rb
require 'uri'
require 'net/http'

url = URI("http://mockbin.com/har")

http = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Post.new(url)
request["content-type"] = 'application/x-www-form-urlencoded'
request.body = "foo=bar&hello=world&hello%20world=world%20hello"

response = http.request(request)
puts response.read_body

File: test/fixtures/output/javascript/fetch/application-form-encoded.js
fetch("http://mockbin.com/har", {
  "method": "POST",
  "headers": {
    "content-type": "application/x-www-form-urlencoded"
  },
  "body": {
    "foo": "bar",
    "hello": "world",
    "hello world": "world hello"
  }
})
.then(response => {
  console.log(response);
})
.catch(err => {
  console.error(err);
});
